<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDaDashboardInteraction</class>
 <widget class="QWidget" name="C_SyvDaDashboardInteraction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C_SyvDaDashboardInteraction</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,0,0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins" name="pc_LabelConnectText">
     <property name="text">
      <string>TConnect</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButtonConnect">
     <property name="minimumSize">
      <size>
       <width>42</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>42</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgeGbxTransparent" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="pc_LabelConnectStatus">
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>3</width>
       <height>3</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="stw::opensyde_gui_elements::C_OgePubSvgIconOnly" name="pc_PushButtonDarkMode">
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string>Dark</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeGbxTransparent</class>
   <extends>QGroupBox</extends>
   <header>C_OgeGbxTransparent.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabGenericNoPaddingNoMargins</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGenericNoPaddingNoMargins.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgePubSvgIconOnly</class>
   <extends>QPushButton</extends>
   <header>C_OgePubSvgIconOnly.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
